oauth2client/client.py

```diff
@@ -1222,6 +1227,31 @@ def create_scoped(self, scopes):
         """
         return self
 
+    @classmethod
+    def from_json(cls, s):
+        from oauth2client.service_account import _ServiceAccountCredentials
```

Comment:
So there should be a comment here referencing the issue filed to resolve the circularity, at least?